README.md

```diff
@@ -32,30 +31,43 @@
 
 ## Introduction
 
-If you’ve ever worked or considered working for a startup or fast-growing tech company, you probably have experienced or tried to learn about stock options, RSUs, and other types of **equity compensation**.
+Growing numbers of people are working in startups and fast-growing tech companies. If you are one of them, or are considering becoming one, you probably have experienced or tried to learn about stock and stock options, RSUs, and other types of [**equity compensation**](http://www.investopedia.com/terms/e/equity-compensation.asp).
```

Comment:
Let's get a number here to help us establish significance. How much is the field "growing"? (We know it's a lot! And we also know (I think) that there is an increasing number of non-tech people (people more likely to be unfamiliar with this material) in the field. Calling that out would help the draw.) (A couple of links: startups created [2.5 million jobs ](https://www.census.gov/newsroom/press-releases/2017/business-dynamics.html) in 2015 alone; K[auffman Index 2017](https://www.kauffman.org/kauffman-index/reporting/startup-activity) tracks startup activity.)